jwst/saturation/x_irs2.py

```diff
@@ -30,37 +30,41 @@
 
 
 def _get_irs2_parameters(input_model, n=None, r=None):
-    """Get the parameters describing IRS2 readout format.
+    """
+    Get the parameters describing IRS2 readout format.
 
     Parameters
     ----------
-    input_model: most likely a RampModel object
+    input_model : DataModel
+        This model is most likely a RampModel object
         This is used for getting the width of the reference output and
         the values of NRS_NORM and NRS_REF.
 
-    n: int or None (default is None)
+    n : int or None
+        Default is None.
```

Comment:
no need to put the defaults here because they are clear from the function definition.  And if this ever gets picked up by our ReadtheDocs build, it will be smart enough to show those defaults without the need to re-write them here.

Reply:
I think these comments may have been missed

Reply:
ok something strange is going on I made all those changes a few days ago. My branch looks updated. 
Let me figure out what is going on